src/planscape/planning/tests/test_scenario_views.py

```diff
@@ -273,16 +273,7 @@ def test_create_scenario_viewer_user(self):
             payload,
             content_type="application/json",
         )
-        self.assertEqual(response.status_code, 200)
-        output = response.json()
-        scenario_id = output["id"]
-        self.assertEqual(Scenario.objects.count(), 1)
-        self.assertEqual(ScenarioResult.objects.count(), 1)
-        scenario = Scenario.objects.get(pk=scenario_id)
-        self.assertEqual(scenario.planning_area.pk, self.planning_area.pk)
-        self.assertEqual(scenario.configuration, self.configuration)
-        self.assertEqual(scenario.name, "test viewer scenario")
-        self.assertEqual(scenario.user, self.viewer_user)
+        self.assertEqual(response.status_code, 403)
```

Comment:
is this change correct? Seems like the test should be checking the successful case not a 403

Reply:
Should someone with a Viewer role have `add_scenario` permissions on a Planning Area?
According to the spec and our [permission migration](https://github.com/OurPlanscape/Planscape/blob/6b72c9f5c2b1d3ef9d9bdccbe8ad15b5b826e88d/src/planscape/collaboration/migrations/0002_add_permissions.py#L14), only collaborators and owners/creators should have this.

Reply:
true! thanks for clarifying, I misread the test title